apps/heft/src/plugins/TypeScriptPlugin/Tslint.ts

```diff
@@ -36,28 +36,63 @@ export class Tslint extends LinterBase<TTslint.RuleFailure> {
     this._fileSystem = options.fileSystem;
   }
 
+  private static _resolvePackageMainFilePath(packagePath: string): string {
```

Comment:
How is this appreciably different from `Import.resolveModule`?

Reply:
IIRC the config file paths that were earlier returned by resolveModule were resulting in paths that were causing an error when we then tried to call fileSystem.readFile on them. It wasn't clear to me from the [documentation ](https://github.com/microsoft/rushstack/blob/1f3a8bd81c79e975ca55ec41f24011aae9839461/libraries/node-core-library/src/Import.ts#L181) of resolveModule why this was happening, I thought it was returning folder names but looking at the documentation again it could be ommitting file extensions?

Reply:
It seems to be working since the change to referencing Import, so I removed the ad hoc function